### PR TITLE
feat: add documentation

### PR DESCRIPTION
**Pull request by benpreiser**

### CODE REVIEW

File: EOS.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    
+
+       { "compression": "",
+        "packed_trx": "",
+        "signatures": [
+            "",
+            ""
+        ]
+       }
```
Comment (lifeiscontent) [inline EOS.md:44]: use json syntax highlighting, and format each document in VSCode.

File: README.md [hunk 2]
```diff
@@ -1,73 +1,40 @@
...
+      |  XRP  | Yes  | Yes  |
+      |  ZEC  | Yes  | No  |
 
-```sh
-npm create electron-vite
-```
+## Installing and Downloading
 
-![electron-vite-react.gif](https://github.com/electron-vite/electron-vite-react/blob/main/public/electron-vite-react.gif?raw=true)
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```
Comment (lifeiscontent) [inline README.md:30]: this project was not bootstrapped with Create React App. please read over your changes when you copy/paste stuff please.

Comment (lifeiscontent) [reply README.md:30]: https://github.com/electron-vite/electron-vite-react this is what we used.

File: README.md [hunk 3]
```diff
@@ -1,73 +1,38 @@
...
-🔩 Supports C/C++ native addons  
-🐞 Debugger configuration included  
-🖥 Easy to implement multiple windows
+Withdraw Funds without BitGo:
 
-## 🛫 Quick start
+- Unsigned Sweep: Build an unsigned transaction using the user and backup public keys, independent from any BitGo services.
+- Non-BitGo Recoveries: Build a transaction using the recovery KeyCard, independent from any BitGo services.
+- Coin Support:
+  | | Production | Test |
```
Comment (lifeiscontent) [inline README.md:12]: run the formatter on ALL the markdown files.

**Review by arnold-yip — APPROVED** [review]

**Pull request merged**